mltrace/cli/cli.py

```diff
@@ -94,7 +94,8 @@ def show_info_card(run_id: int, count: int = None, num_outputs: int = None):
     dependencies = (
         " ".join(cr_info.dependencies) if cr_info.dependencies else "None"
     )
-    click.echo(f"└─Dependencies: {dependencies}")
+    click.echo(f"├─Dependencies: {dependencies}")
+    click.echo(f"└─Test Results: {cr_info.test_result}")
```

Comment:
print with indentation -- for example, `json.dumps(cr_info.test_result, indent=2)`